MultiEditUtils.sublime-settings

```diff
@@ -1,3 +1,7 @@
 {
-  "live_split_selection" : true
+  "live_split_selection" : true,
+  // whether the tab key should jump to the next field during the selection field mode
+  "selection_fields_tab_enabled": true,
+  // whether the escape key should cancel the selection field mode
+  "selection_fields_escape_enabled": true

```

Comment:
If these options are set to `false` the corresponding keys are disabled. Since `tab` and `escape` are overloaded with a lot of contexts, I think some users might want to disable them for this context without touching other keybindings.
I could imagine some people wants to disable the `tab` key in favor to increase the indentation instead of jumping to the next field.

Going to the next field would still be possible with the normal keybinding (e.g. `alt+d`) and it the fields are "popped" after the last field.